Sources/Routing/Route.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+import HTTP
+
+/// A representation of a `route` or endpoint in your application
+/// It is made up of the basic components of a `route` and contains
+/// optional metadata.
+public final class Route {
+    public let host: String
```

Comment:
Is the host guaranteed to be set?

Reply:
It is set here with the same default value that the router uses https://github.com/vapor/routing/pull/34/files#diff-574ab6c0015885711351809459efd45dL20